### PR TITLE
[Issue #7] Detection of KingRoot fails due a typo.

### PR DESCRIPTION
**Pull request by amaa-99**
Fixed a typo on the KingRoot detection code leading the detection to always fail.

### CODE REVIEW

File: app/src/main/java/dev/ukanth/ufirewall/MainActivity.java [hunk 1]
```diff
@@ -2545,7 +2545,7 @@ protected Void doInBackground(Void... params) {
             //open shell if required
             Shell.getShell().isRoot();
             suGranted[0] = Shell.isAppGrantedRoot();
-            unsupportedSU = isSuPackage(getPackageManager(), "com.kingouser.com");
```
Comment (ukanth) [inline app/src/main/java/dev/ukanth/ufirewall/MainActivity.java:2548]: Seems like the package changed. In that case both should be there. Also this PR has other commits. I can't merge.

Comment (amaa-99) [reply app/src/main/java/dev/ukanth/ufirewall/MainActivity.java:2548]: I couldn't find any references to "com.kingouser.com" name by searching in the internet.
The "com.kingroot.kinguser" is also used elsewhere in the afwall code (just make a search for "com.kingroot.kinguser").
If you're sure about "com.kingouser.com", then it seems a check for either would be needed in that location of the code.

Comment (amaa-99) [reply app/src/main/java/dev/ukanth/ufirewall/MainActivity.java:2548]: I'll drop the other commits (there are individual pulls for each of those).

**Review by ukanth — COMMENTED** [review]
Looks good.

**Review by ukanth — APPROVED** [review]

**Pull request merged**